fuel-vm/src/checked_transaction.rs

```diff
@@ -130,13 +130,18 @@ impl<Tx: IntoChecked + UniqueIdentifier> Checked<Tx> {
 }
 
 #[cfg(feature = "test-helpers")]
-impl<Tx: IntoChecked + Default> Default for Checked<Tx>
+impl<Tx: IntoChecked + Default> Checked<Tx>
```

Comment:
You still can implement `Default` trait with default gas price 0 or 1.

Reply:
Maybe we want both? I'm just worried that someone will write a test that needs the gas_price to be parameterized and there will be some hidden gas_price instead.